scripts/deploy.js

```diff
@@ -19,12 +19,15 @@ async function main() {
     340282365167313208607671216367074279424n, // dcfFactor, it mean 0.85 for yearly discount
     1048576, // nonceLimit 1024 * 1024 = 1M samples and finish sampling in 1.3s with IO rate 6144 MB/s: 4k * 2(random checks) / 6144 = 1.3s
     "0x0000000000000000000000000000000000000000", // treasury
-    8192000000000000, // prepaidAmount - ~ 50% discount, 4096 blob cost for 1G data
+    8388608000000000000n, // prepaidAmount - ~ 50% discount, 4096 * 1024 = 4194304 blob cost for 1G data
```

Comment:
for 1TB data?

Reply:
this is for 500G, then let's change to 1T